CLI/test/ressources/CopyInsideTypeDefInsideDependency/input/Copies/YCONVEA.CPY

```diff
@@ -1,6 +1,7 @@
          05  CONVEA-VEA-VEA-OUT-VEA pic X.
          05  CONVEA-VEA-VEA-OUT-VEA-2 pic X.
-         05  CONVEA pic X.
+         05  CONVEA.
+                10 CONVEA-SUB pic X.
```

Comment:
Could you add another sublevel either in this copy or in the other one ?

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1646/commits/5ec741f218377464909aae42e8454ea4de4d1d2e